baselines/dom.generated.d.ts

```diff
@@ -6302,7 +6304,7 @@ declare var HTMLDetailsElement: {
     new(): HTMLDetailsElement;
 };
 
-/** @deprecated - this is not available in most browsers */
+/** @deprecated this is not available in most browsers */
```

Comment:
@orta Do you agree with this change? It seems more like [the standard format](https://jsdoc.app/tags-deprecated.html).

> ```js
> /**
>  * @deprecated since version 2.0
>  */
> function old() {
> }
> ```

Reply:
Yep, the `-` kinda gets put in an odd place in IDEs

![Screen Shot 2021-07-09 at 1 30 51 PM](https://user-images.githubusercontent.com/49038/125078140-e5912b80-e0b9-11eb-97d5-c6381a9dde7c.png)